### DISCUSSION TITLE
Test Courses

### DISCUSSION THREAD

**Discussion by nathancoooper** [kantord/LibreLingo#2187] [Ideas]
I'm not sure how this could be integrated but it would be great if, as you were building a course (for example, I'm currently working on the French from English course), you could have that course linked to your LibreLingo account as the creator and could, then, test the course before it is released to the general public.

**Comment by kantord** [comment]
This is more or less possible, albeit in a way that's much more simplified technologically and requires some manual help on my part.

Basically, I can set up a repo for you under the LibreLingo organizatino and allow you to edit it. Then I can set up for this course to be deployed from this repo but not linked anywhere publicly. There are "secret" pages for course authors where your course will be available, and knowing the link you will be able to test it, but it won't be "officially released" or linked anywhere on the main page

**Reply by nathancoooper** [reply]
Ahhh, okay; this sounds perfect. I'll write back to you in a month-or-so when I'm ready to begin course testing!
